### PR TITLE
Extended serialization to have negative stride mean 'only save last'

### PR DESCRIPTION
**Pull request by vkip**
.. and also allow loading from different file than saving to.

### CODE REVIEW

**Comment by vkip** [comment]
jenkins build this please

File: opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp [hunk 1]
```diff
@@ -264,9 +283,14 @@ class SimulatorFullyImplicitBlackoilEbos
...
         EWOMS_REGISTER_PARAM(TypeTag, std::string, SaveFile,
-                             "FileName for .OPMRST file used for serialized state. "
+                             "FileName for .OPMRST file used for saving serialized state. "
                              "If empty, CASENAME.OPMRST is used.");
         EWOMS_HIDE_PARAM(TypeTag, SaveFile);
+        EWOMS_REGISTER_PARAM(TypeTag, std::string, LoadFile,
+                             "FileName for .OPMRST file used to load serialized state. "
+                             "If empty, CASENAME.OPMRST is used.");
+        EWOMS_HIDE_PARAM(TypeTag, LoadFile);
+
```
Comment (akva2) [inline opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp:293]: skip this blank line. and please update SaveStep documentation string.

**Review by akva2 — APPROVED** [review]
One minor cosmetic and missing doc string update.

**Comment by blattms** [comment]
would be cool if we would also update the documentation of SaveStep` in opm/simulators/flow/SimulatorFullyImplicitBlackoilEbos.hpp accordingly.

**Comment by akva2** [comment]
jenkins build this please

**Pull request merged**